src/main/java/frc/robot/commands/Climb.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.commands;
+
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+import java.util.function.Supplier;
+
+public class Climb extends CommandBase {
```

Comment:
IMO we should consider creating an automatic climb when we are not the only robot